extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -308,6 +311,10 @@ md-card.conversation-skin-supplemental-card {
   margin-top: 2px;
   width: 46px;
 }
+.conversation-skin-responses-animate-slide{
+  padding-right: 56px;
+  width: 528px;
```

Comment:
Initially .conversation-skin-responses-animate-slide was inheriting a "width: 472px" property from md-card.conversation-skin-tutor-card that is set in the css file. Because the container is now being shifted to the left 56px to fully wrap the avatars the width property needed to be increased by 56px as well so the response text would wrap properly.  I overlooked that .conversation-skin-responses-animate-slide also has a "max-width: 472px" property to prevent the type of wrapping issue you are describing.  I believe adding a "max-width: 528px" property to .conversation-skin-responses-animate-slide would fix the bug.  Does this make sense or should i go in a different direction?